src/modules/endpoint/endpoints/feedback/categoriesFeedback.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+
+import { apiUrl } from '../../constants'
+import ParamMissingError from '../../errors/ParamMissingError'
+import FeedbackEndpointBuilder from '../../FeedbackEndpointBuilder'
+
+const CATEGORIES_FEEDBACK_ENDPOINT_NAME = 'categoriesFeedback'
+const POSITIVE_RATING_STRING = 'up'
+const NEGATIVE_RATING_STRING = 'down'
+
+export default new FeedbackEndpointBuilder(CATEGORIES_FEEDBACK_ENDPOINT_NAME)
+  .withParamsToUrlMapper((params): string => {
```

Comment:
params should have types imo. If you do not add types flow interpretes that params has the type:

`{ city: string, language: string }`

In the second mapper blow more types like `isPositiveRating` get added. This makes it not obvious which type params has.